tools/serverpod_vscode_extension/src/extension.ts

```diff
@@ -13,7 +13,7 @@ let client: LanguageClient;
 export function activate(context: ExtensionContext) {
 	const serverOptions: ServerOptions = {
 		command: 'serverpod',
-		args: ['language-server', '--no-development-print'],
+		args: ['language-server', '--quiet'],
```

Comment:
Note that this is a change to the extension - Do we need to release this change somewhere or is it OK just to change here for now?

Reply:
It's ok to change as the change is only happening on the `main` branch and not to a released version of Serverpod.